models/public/ssd_resnet50_v1_fpn_coco/model.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "ssd_resnet50_v1_fpn_coco" model is a SSD FPN object detection architecture based on Resnet50 (https://arxiv.org/abs/1708.02002).
+  The model has been trained from the Common Objects in Context (COCO) image dataset.
+  Model repository: https://github.com/tensorflow/models/blob/master/research/object_detection
```

Comment:
We try to give the same model description as in `.md` files, except the links format differs, see https://github.com/opencv/open_model_zoo/blob/develop/models/public/densenet-121-caffe2/model.yml cc @eizamaliev

Reply:
Done